File: frontend/src/types/place.ts
export interface Place {
  id: string
  name: string
  category: string // restaurant | cafe | fast_food
  types: string[]
  rating: number | null
  user_ratings_total: number | null
  price_level: number | null // 0 (ücretsiz) – 4 (çok pahalı)
  address: string | null
  lat: number
  lon: number
  distance_m: number
  open_now: boolean | null
  photo_name: string | null
}

File: backend/app/schemas/place.py
from pydantic import BaseModel


class Place(BaseModel):
    """Bir mekanı (restoran/kafe/fast-food) temsil eden veri modeli."""

    id: int
    name: str
    category: str  # restaurant | cafe | fast_food
    cuisine: str | None = None
    address: str | None = None
    lat: float
    lon: float
    distance_m: int  # kullanıcıya kuş uçuşu mesafe (metre)
